CHANGELOG/CHANGELOG-3.1.md

```diff
@@ -1,7 +1,25 @@
-# Release notes for v3.1.0
+# Release notes for v3.1.1
 
 [Documentation](https://kubernetes-csi.github.io)
 
+# Changelog since v3.1.0
+
+## Changes by Kind
+
+### Bug or Regression
+ - Storage capacity: managed-by label potentially too long with a long node name and enableNodeDeployment set to true ([#739](https://github.com/kubernetes-csi/external-provisioner/pull/739), [@akalenyu](https://github.com/akalenyu))
```

Comment:
so this is the only critical bug we want to fix in a patch release? Just want to understand how we pick which one to include or not.

Reply:
A user asked for it. There is no systematic triaging of bugs and no automatic backporting of bugs. 

From https://kubernetes-csi.github.io/docs/project-policies.html (emphasis mine):
> Every minor release branch will be supported with patch releases on an *as-needed* basis for at least 1 year